### ISSUE TITLE
Add option to `--replace` cloud resources if they already exist

### ISSUE THREAD

**Issue by salsferrazza** [GoogleCloudPlatform/market-data-transcoder#3]
With the `--replace` option, the transcoder will not try to use existing cloud resources that correspond to message types in the schema (schemas, topics, BigQuery tables), rather it will delete and replace them with new ones derived from the schema represented in `--schema_file`. This would be a destructive action, so would have to be explicit and used with care.

**Comment by mservidio** [comment]
@salsferrazza Would this apply for files too?

**Comment by salsferrazza** [comment]
Yes so truncating the existing POSIX data and schema files if they exist On Dec 29, 2022 13:11, Mark Servidio ***@***.***> wrote:
@salsferrazza Would this apply for files too?
